components/rabbitmq/config/rabbitmq-env.conf

```diff
@@ -0,0 +1 @@
+NODENAME=cloudify-manager@localhost
```

Comment:
Without it, deploying the manager image with a different hostname to the one it had originally was causing rabbit to fail to start because it couldn't find the node (default is something like rabbit@<hostname>, and <hostname> was different.)